prow/config.yaml

```diff
@@ -2920,3 +2920,40 @@ periodics:
         path: /mnt/disks/ssd0
       name: cache-ssd
 
+- interval: 24h
+  name: ci-kubernetes-pull-gce-federation-deploy
+  spec:
+    containers:
+    - args:
+      - --timeout=90
+      - --bare
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/service-account/service-account.json
+      - name: USER
+        value: prow
+      - name: JENKINS_GCE_SSH_PRIVATE_KEY_FILE
```

Comment:
I don't this think job needs SSH access to VMs. Do we need it for any non-GKE Kubernetes jobs?